tests/unit/tasks/server/livereload-server-test.js

```diff
@@ -75,6 +75,16 @@ describe('livereload-server', function() {
           preexistingServer.close(done);
         });
     });
+
+    it('starts with custom host', function() {
+      return subject.start({
+        liveReloadHost: '127.0.0.1',
```

Comment:
if anybody has a suggestion for a better host that will always work, let me know.